scripts/run_eks_smoke_test.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+BUILD=${TRAVIS_BUILD_NUMBER:-local}
+USE_REGION=${USE_REGION:-us-east-1}
+
+handle_error() {
+    trap - EXIT
+    show_kube_info || true
+    cleanup || true
+}
+
+cleanup() {
+    delete_kube_resources
+}
+
+delete_kube_resources() {
+    kubectl delete -f $SCRIPT_DIR/eks-smoke-test/kip-nginx.yaml
+    kubectl delete -f $SCRIPT_DIR/eks-smoke-test/regular-node-nginx.yaml
+    kubectl delete -n kip-smoke-tests pod test --ignore-not-found
+    kubectl delete pod test --ignore-not-found
+}
+
+show_kube_info() {
+    kubectl get pods -A
+    kubectl get nodes
+    kubectl -n kip-smoke-tests describe pod -l statefulset.kubernetes.io/pod-name=kip-build-kip-provider-0
+    kubectl logs -n kip-smoke-tests -l statefulset.kubernetes.io/pod-name=kip-build-kip-provider-0r -c init-cert --tail=-1
+    kubectl logs -n kip-smoke-tests -l statefulset.kubernetes.io/pod-name=kip-build-kip-provider-0 -c kip --tail=-1
+}
+
+update_vk() {
+    local version="$(git describe)"
+    local patch_init="{\"spec\":{\"template\":{\"spec\":{\"initContainers\":[{\"image\":\"elotl/init-cert:$version\",\"name\":\"init-cert\"}]}}}}"
```

Comment:
That's a lot of of back-slahes escapes.

I'd use single quotes to avoid this:
```'{"spec":{"template":{"spec":{"initContainers":[{"image":"elotl/init-cert:","name":"init-cert"}]}}}}'```

Reply:
My shellcheck is complaining about [this](https://github.com/koalaman/shellcheck/wiki/SC2016) if I use single quotes.